repo.groovy

```diff
@@ -30,6 +30,11 @@ repos = [
     workflowComponent: true,
     chart: 'fluentd'],
 
+  [ name: 'jenkins-node',
```

Comment:
Certainly! There's no Makefile in the project at the moment but I can cook one up in a sec. :)